authors/settings/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .base import *
+from .production import *
+
+try:
+    from .local import *
```

Comment:
can you explain this line, because I think we neglected the `local` file and we agreed on the `development` file, why am I seeing local?

Reply:
@huxaiphaer and @collinewait, don't let the word "local" confuse you. If you check the files, you'll notice that we pushed only base and production, therefore having local.py for our settings to work only for our local machines.

Reply:
`local` and `development` are the same , we had agreed with @inno-asiimwe , to use `development`, so that's why i wanted you to be clear on this. Because remeber i already documented in my previous PR on README that we are to use development file for our local machines.